_posts/2016/2016-11-02-search-technology-meetup.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: post
+title: "Interesting insights from the Search Technology Meetup VI"
+date: "2016-11-02 09:00:00"
+image: blog-header/elastic-meetup.jpg
+categories: events
+authors: ["Rose"]
+---
+
+<style>
+.twitter-tweet {
+   margin: auto;
+}
+</style>
+
+Last friday, we hosted the seventh edition of the Search Technology Meetup Hamburg organized by [Shopping24](https://www.shopping24.de) in our new building in Holstenwall 5.
+The room was packed and participants from all kinds of backgrounds engaged in discussions with the invited speakers even before the presentations started.
+Next to some snacks and refreshing drinks, we then saw two brilliant talks on search architecture and how to use statistical as well as machine learning models for improving search.
+Therefore, [Miguel Olivares](https://twitter.com/moliware) and Christian Würtz from [Xing](https://www.xing.com/) as well as Data Scientist [Andrew Clegg](https://twitter.com/andrew_clegg) from [Etsy](https://www.etsy.com/) gave us interesting insights into their search technology infrastructure, respectively.
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">The room is packed and listening to <a href="https://twitter.com/andrew_clegg">@andrew_clegg</a>! The <a href="https://twitter.com/hashtag/search?src=hash">#search</a> community is in fire in <a href="https://twitter.com/hashtag/hamburg?src=hash">#hamburg</a> at <a href="https://twitter.com/hashtag/stmhh?src=hash">#stmhh</a>. Thanks for joining us! <a href="https://t.co/1j402flFF7">pic.twitter.com/1j402flFF7</a></p>&mdash; Search Tech Meetup (@stmhamburg) <a href="https://twitter.com/stmhamburg/status/792067869114851328">October 28, 2016</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+## Evolution of search at Xing
```

Comment:
This section looks too much like a "block".
Maybe you can add a paragraph inbetween, where it matches?